packages/core/src/fetchers/fetchAllEntities.ts

```diff
@@ -43,7 +43,7 @@ export const fetchAllEntries = async ({
       };
     }
 
-    const query = { 'sys.id[in]': ids, locale, limit, skip };
+    const query = { 'sys.id[in]': ids, locale, limit, skip, include: 2 as 2 | undefined };
```

Comment:
What do you think about using `2 as const` here instead?
```suggestion
    const query = { 'sys.id[in]': ids, locale, limit, skip, include: 2 as const };
```

Reply:
Since the 'include' prop on the query param used a numeric literal, this is what I did to make TS happy as it wouldn't just let me pass in a number to it.